paper-slider.html

```diff
@@ -90,6 +90,11 @@
       :host(:focus) {
         outline: none;
       }
+        
+      :host-context([dir="rtl"]) {
```

Comment:
@dschuyler One more thing - I forgot about the `editable` option (which adds a text field at the end of the slider). This selector should be `:host-context([dir="rtl"]) #sliderContainer` so that we don't flip the text field as well.